### DISCUSSION TITLE
Provide reactive notebooks in Quarto

### DISCUSSION THREAD

**Discussion by parmsam** [quarto-dev/quarto-cli#6941] [Feature Requests]
### Description

Consider adding reactive notebook similar to [reactor](https://github.com/herbps10/Reactor/) in R, [ipyflow](https://github.com/ipyflow/ipyflow) in Python, or [Pluto.jl](https://github.com/fonsp/Pluto.jl) in Julia. Could be a special mode similar to the visual editor for Quarto. Would make relevant chunks update based on data changes in the notebook, making experimentation easier without the need to re-excute previous lines of code.

**Comment by mcanouil** [comment]
From what I understand `reactor` requires an R session since they can be run via shiny, it is already supported, see <https://quarto.org/docs/interactive/shiny/>.

`ipyflow` is a Jupyter kernel, thus already supported, see <https://quarto.org/docs/computations/python.html#kernel-selection>.

Pluto.jl is not compatible with Jupyter, it is an alternative, so unlikely Julia supports will be in two different system.
